extensions/amp-form/0.1/amp-form.js

```diff
@@ -1045,21 +1045,21 @@ export class AmpForm {
             .then(rendered => {
               rendered.id = messageId;
               rendered.setAttribute('i-amphtml-rendered', '');
-              container.appendChild(rendered);
               const renderedEvent = createCustomEvent(
                   this.win_,
                   AmpEvents.DOM_UPDATE,
                   /* detail */ null,
                   {bubbles: true});
               container.dispatchEvent(renderedEvent);
+              this.resources_.mutateElement(container, () => {
```

Comment:
Should we also wait to dispatch DOM_UPDATE event?

Reply:
done. Strangely, the check type run fails when the dispatch event is inside the mutateElement call. 

Compiler issues for check-types.js:
extensions/amp-form/0.1/amp-form.js:1048: ERROR - actual parameter 1 of Resources$$module$src$service$resources_impl.prototype.mutateElement does not match formal parameter
found   : (Element|null)
required: Element
              this.resources_.mutateElement(container, () => {
                                            ^^^^^^^^^

I added a check type supress annotation to bypass it.

Reply:
Use `devAssert(container)` instead.

Reply:
done. thanks